modules/ipi-install-setting-cluster-node-hostnames-dhcp.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+//  *installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ipi-install-setting-cluster-node-hostnames-dhcp_{context}"]
+= Setting the cluster node hostnames through DHCP
+
+On {op-system-first} machines, the hostname is set through NetworkManager. 
+By default, the machines obtain their hostname through DHCP. 
```

Comment:
```suggestion
By default, the machines obtain their hostnames through DHCP. 
```

Reply:
It was tough to navigate the singular vs plural use of hostnames in this section. I would maybe pick one and try to stick with it for consistency.

Reply:
Good suggestion. I changed all but one to plural. In one case the singular just makes more sense.